sniffles/ruletrafficgenerator.py

```diff
@@ -1631,95 +1638,165 @@ def clear_globals(self):
         global MAC_IP_MAP
         global VENDOR_MAC_DIST_DOMAIN
         global VENDOR_MAC_DIST
+        global VENDOR_MAC_DIST_OPTION
+
+        global VENDOR_MAC_DIST_RESET
```

Comment:
lets get rid of VENDOR_MAC_DIST_RESET and instead test for presence of vendor_mac_dist.